backend/tests/integration/records/serializers/json/test_experiments.py

```diff
@@ -94,186 +94,23 @@ def test_experiments_json_search(inspire_app, datadir):
     assert expected_updated == response_updated
 
 
-def test_experiments_detail(inspire_app, datadir):
```

Comment:
was all of this removed on purpose?

Reply:
Ouch, this looks like my error during resolving conflict. Thanks for spotting this :+1: